packages/next/build/webpack-config.ts

```diff
@@ -273,7 +273,7 @@ export default async function getBaseWebpackConfig(
   }
 
   function getReactProfilingInProduction() {
-    if (config.reactProductionProfiling) {
+    if (process.argv.length > 2 && process.argv.includes('--profile')) {
```

Comment:
This should be passed in through the functions instead of depending on `process.argv` which can be wrong

Reply:
The least intrusive way seemed to be what I did in 69002e8,
If this doesn't seem right I can alternatively pass it in the conf variable instead of adding an extra parameter but then I'd have to modify [loadConfig](https://github.com/vercel/next.js/blob/canary/packages/next/next-server/server/config.ts#L210-L217) since it ignores the config a user defines in the file if customConfig is not null